tools/tools.qmd

```diff
@@ -73,21 +76,16 @@ A curated collection of tools and interfaces to help you work effectively with S
-| **Sublime Text**       | [**SublimeStan**](https://github.com/djsutherland/sublime-stan) | Syntax highlighting for Stan in Sublime Text. |
-| **JupyterLab**         | [**jupyterlab-stan-highlight**](https://www.npmjs.com/package/jupyterlab-stan-highlight) | Syntax highlighting for Stan code blocks in JupyterLab. |
-| **Markdown Editors**   | **Pandoc**, **Pygments** | Stan syntax highlighting in markdown via [Pandoc](https://pandoc.org/) and [Pygments](https://pygments.org/). |
-| **LaTeX**              | [**lstbayes**](https://github.com/jrnold/lstbayes) | LaTeX listings for Stan syntax highlighting. |
-
----
-
-## Additional Notes
-
-- **Recommendation for New Users**: We suggest starting with **CmdStanR** (for R users) or **CmdStanPy** (for Python users) along with high-level interfaces like **brms** or **rstanarm** for ease of use.
-
+| **RStudio**            | **Built-in Support** | RStudio 1.2+ includes Stan syntax highlighting and code snippets.  [(Source code)](https://github.com/rstudio/rstudio/blob/main/src/gwt/acesupport/acemode/stan_highlight_rules.js){target="_blank"} |
+| **Visual Studio Code** | [**Stan VSCode Extension**](https://marketplace.visualstudio.com/items?itemName=stan-dev.stan-vscode){target="_blank"} | Stan language support with syntax highlighting, autocompletion, and snippets. |
+| **Emacs**              | [**stan-mode**](https://github.com/stan-dev/stan-mode){target="_blank"} | Major mode for Stan with syntax highlighting and indentation. |
+| **Vim**                | [**stan-vim**](https://github.com/eigenfoo/stan-vim){target="_blank"} | Syntax highlighting, indentation, and code folding for Stan in Vim. |
+| **Atom**               | [**language-stan**](https://github.com/stan-dev/atom-language-stan){target="_blank"} | Stan language support in Atom editor. |
+| **Sublime Text**       | [**SublimeStan**](https://github.com/djsutherland/sublime-stan){target="_blank"} | Syntax highlighting for Stan in Sublime Text. |
+| **JupyterLab**         | [**jupyterlab-stan-highlight**](https://www.npmjs.com/package/jupyterlab-stan-highlight){target="_blank"} | Syntax highlighting for Stan code blocks in JupyterLab. |
+| **Javascript**         | [**Prism**](https://prismjs.com/){target="_blank"} </br> [**Highlight.js**](https://github.com/highlightjs/highlight.js){target="_blank"} | Lightweight syntax highlighting library - [(Source code)](https://github.com/PrismJS/prism/blob/master/components/prism-stan.js){target="_blank"}. </br> Syntax highlighter written in javascript - [(Source code)](https://github.com/highlightjs/highlight.js/blob/main/src/languages/stan.js){target="_blank"} |
+| **Markdown Editors**   | [**Pandoc**](https://pandoc.org/){target="_blank"} </br> [**Pygments**](https://pygments.org/){target="_blank"} | Stan syntax highlighting in markdown. [(Source code)](https://github.com/pygments/pygments/blob/master/pygments/lexers/modeling.py){target="_blank"} |
```

Comment:
The upstream source for pandoc is https://invent.kde.org/frameworks/syntax-highlighting/-/blob/master/data/syntax/stan.xml

Reply:
thanks for this - added.